deploy/charts/jetstack-agent/tests/values/custom-volumes.yaml

```diff
@@ -0,0 +1,11 @@
+volumes:
+  - name: cabundle
+    configmap:
+      name: cabundle
+      optional: true
```

Comment:
Why is this optional? Isn't this prone to silent unexpected situations such as you forget to create the CA bundle configmap and the pod somehow isn't able to connect?